bigchaindb/exceptions.py

```diff
@@ -1,65 +0,0 @@
-"""Custom exceptions used in the `bigchaindb` package.
```

Comment:
Valid point, lets handle (as you proposed) in this ticket: https://github.com/bigchaindb/bigchaindb-common/issues/23